buildstrap

```diff
@@ -46,7 +46,14 @@ else
         FILE=`cat $OUT_TMP/var/lib/apt/lists/*Packages | grep -e "^Filename:.*/${PACKAGE}_" | cut -d' ' -f2`
         echo "Fetching ${MIRROR}${FILE}..."; echo ""
         curl -L ${MIRROR}${FILE} --output $TDIR/`basename $FILE`
-        $root dpkg-deb -x $TDIR/`basename $FILE` $OUT_TMP; done; fi
+        $root dpkg-deb -x $TDIR/`basename $FILE` $OUT_TMP; done
+
+    # FIXME: Install python-scapy manually because it comes from universe and
+    #        we need to use a version that is not in 18.04 due to a bug.
+    FILE=pool/universe/s/scapy/python-scapy_2.4.0-1_all.deb
+    echo "Fetching ${MIRROR}${FILE}..."; echo ""
+    curl -L ${MIRROR}${FILE} --output $TDIR/`basename $FILE`
+    $root dpkg-deb -x $TDIR/`basename $FILE` $OUT_TMP; fi
```

Comment:
I prefer package specific changes/scripts go into their package files into the packages/ directory. Probably they can be functions that are called and that follow a naming convention like `do_<package-name>_post()` ? So the python specific one would be called, do_python2_post(). If they are undefined for a package, then we do nothing.
I really don't want package specific things in `buildstrap`.